drivers/dac/ad5770r/ad5770r.h

```diff
@@ -319,7 +323,10 @@ struct ad5770r_dev {
 struct ad5770r_init_param {
 	/* SPI */
 	spi_init_param				spi_init;
-
+	/* GPIO */
+	struct gpio_init_param			*nalarm;
```

Comment:
You should give a hint some where that these gpios are optional. At least in the naming, but I think is better in a commnet.

Reply:
Will add a comment.